src/cloudgraphs/MultisessionUtils.jl

```diff
@@ -151,7 +155,7 @@ Return Dict{Symbol, Int} of vertex symbol to Neo4j node ID of MULTISESSION const
 """
 function findExistingMSConstraints(fgl::FactorGraph)
   loadtx = transaction(fgl.cg.neo4j.connection)
```

Comment:
we should use executeQuery! function for these kind of calls instead.  I already wrote a executeQuery somewhere.  The issue `commit` call for healthy Neo4j.jl use.  The open ports if you don't issue.

Reply:
https://github.com/JuliaRobotics/Caesar.jl/blob/5784e15473bee66a9fb858ae16502ef50c6239c5/src/cloudgraphs/CloudGraphIntegration.jl#L50

Reply:
will fix executeQuery separately -- see issue #150

Reply:
Should all be done now, I think